python/tests/integration/test_max_param_length_env_server_integration.py

```
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+import pytest
+
+pytestmark = pytest.mark.server_integration
+
+
+def _make_project(make_server_project):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_make_project`

(ANN202)